velox/common/file/FileSystems.cpp

```diff
@@ -86,7 +86,7 @@ class LocalFileSystem : public FileSystem {
 
   std::unique_ptr<ReadFile> openFileForRead(
       std::string_view path,
-      const FileOptions& /*unused*/) override {
+      const FileOptions& options) override {
```

Comment:
You still need to put 'options' in comments to avoid linter warning as unused parameter.

Reply:
Updated, thanks!